db/data_migration/20170810111203_update_world_news_stories_in_rummager.rb

```diff
@@ -0,0 +1,21 @@
+world_news_story_type_id = 4
+
+editions = Edition
```

Comment:
Could you use `NewsArticle.by_subtype(NewsArticleType::WorldNewsStory)` here as well? Probably not a biggie as I assume the outcome would be the same